content/docs/user-guide/components/reference/atom/hdri-skybox.md

```diff
@@ -1,20 +1,38 @@
 ---
+linkTitle: HDRi Skybox
 title: HDRi Skybox Component
-linktitle: HDRi Skybox
-description: 'Open 3D Engine (O3DE) HDRi Skybox component reference.'
+description: Create a skybox using the HDRi Skybox component in your Open 3D Engine (O3DE) project. 
 toc: true
 ---
 
-The **HDRi Skybox** component creates a skybox with an HDR image in your scene. 
+The **HDRi Skybox** component creates a skybox with a high dynamic range (HDR) image in your scene.
 
 
-## Provider ##
+## Provider
 
 [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
 
 
+## Limitations
+
+The HDRi Skybox and **Physical Sky** components are interchangeable skybox solutions. If you have both components or multiples of one component, only the first active skybox will render.
+
+
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
+![HDRi Skybox interface](/images/user-guide/components/reference/atom/hdri-skybox-component-ui.png)
+
+
+### HDRi Skybox properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Cubemap Texture** | A *cubemap texture* is a collection of six 2D images. Each image is a square of equal size and maps to a face of a cube. The cubemap represents the sky in your scene at every direction of the world axes. | A `.streamingimage` product asset. |  |
+| **Exposure** | Exponentially scales the intensity of the exposure. | Float: -20.0 - 20.0 | `0.0` |
```

Comment:
HDRi Skybox exposure value unit is a EV100 scale (a Log2 exponential luminance and exposure value table used in photography and film), which is an [Exposure Value (EV)](https://en.wikipedia.org/wiki/Exposure_value) with a 100 International Standards Organization (ISO).  Within the Atom rendering pipeline, the skybox pixel intensity is multiplied by the EV100 luminance value to generate a HDR value to be rendered. The final on screen HDR values will be altered by render passes that occur later in the rendering pipeline, such as the 'Exposure Component' (a fullscreen postfx pass), and then the Display Mapper which will tonemap the pixel and apply the ODT (output display transform) resulting in final value to be rendered on screen (to either a LDR or HDR monitor.)